plugins/builddep.py

```diff
@@ -191,8 +191,9 @@ def _spec_deps(self, spec_fn):
         try:
             spec = rpm.spec(spec_fn)
         except ValueError as ex:
-            msg = _("Failed to open: '%s', not a valid spec file: %s") % (
-                    spec_fn, ex)
+            self.rpm_error_occured = True
+            msg = _("Failed to open: '%s', not a valid spec file: %s.") % (
+                    spec_fn, str(ex).strip())
```

Comment:
FWIW I'd change the message string to:
```
"Failed to open spec file '%s': %s"
```

The "not a valid spec file" is redundant and it's just simpler this way.